src/components/TipCard/TipCard.js

```diff
@@ -5,6 +5,7 @@ import PropTypes from 'prop-types';
 const Tip = ({ id, title, description, mod, rating, date, handleDelete}) => {
   return (
     <article className='tip-card'>
+      <div className={`styling-box mod-${mod}`}></div>
```

Comment:
This is so clean. Great job making this class dynamic so that each module will have a different color on the card.